classification/dataset.py

```diff
@@ -199,18 +198,18 @@ def one_hot(x, num_classes, on_value=1., off_value=0.):
         try:
             # Get necessary variables from annotation
             annotation = self.samples.iloc[index]
-            file_name = annotation[self.config.file_name_col]
+            file_name = annotation[cfg.file_name_col]
             sample_per_sec = self.target_sample_rate
-            frame_offset = int(annotation[self.config.offset_col] * sample_per_sec)
-            num_frames = int(annotation[self.config.duration_col] * sample_per_sec)
+            frame_offset = int(annotation[cfg.offset_col] * sample_per_sec)
+            num_frames = int(annotation[cfg.duration_col] * sample_per_sec)
 
             # Load audio
-            audio = torch.load(os.path.join(self.config.data_path,file_name))
+            audio = torch.load(os.path.join(cfg.data_path,file_name))
         
             if audio.shape[0] > num_frames:
                 audio = audio[frame_offset:frame_offset+num_frames]
             else:
-                print_verbose("SHOULD BE SMALL DELETE LATER:", audio.shape, verbose=self.config.verbose)
+                print_verbose("SHOULD BE SMALL DELETE LATER:", audio.shape, verbose=cfg.verbose)
```

Comment:
what does this error message mean

Reply:
the chunk says its 5 seconds long but the # of samples shows its a bit smaller

Honestly it hasn't broken the code like I expected it to when reviewing this edge case so

Reply:
Yeah that can be covered in the logging pr